docs/advanced-topics/dockstore-cli/yaml-command-line-validator-tool.rst

```diff
@@ -0,0 +1,122 @@
+Yaml Command Line Validator Tool
+================================
+
+Description
+-----------------
+
+The Yaml Command Line Validator Tool is used to verify that a `.dockstore.yml <https://docs.dockstore.org/en/stable/dictionary.html?highlight=.dockstore.yml#dict-dockstore-yml>`_ 
```

Comment:
Links to our own doc should be relative, not absolute. Look for other examples in the doc; here's [one](https://github.com/dockstore/dockstore-documentation/blame/feature/SEAB-4503/CLI-YAML-validation-documentation/docs/index.rst#L33). It's more flexible that way -- if we ever change the the default branch, if we want to fully test a deployed version of your branch, etc.

This comment applies to several lines in the file (I'm not going to flag them all).

Reply:
I have corrected all the links

Reply:
I have added this paragraph about the where to add the `.dockstore.yml` file,

> The Dockstore Yaml Command Line Validator Tool is particularly helpful when you are trying to sync a GitHub repository with Dockstore by adding a [.dockstore.yml](file:///home/fhembroff/Documents/Dockstore/dockstore-documentation/docs/_build/html/dictionary.html#dict-dockstore-yml) to the root or .github directory of the repository. As you can verify that [.dockstore.yml](file:///home/fhembroff/Documents/Dockstore/dockstore-documentation/docs/_build/html/dictionary.html#dict-dockstore-yml) is valid before pushing, and it can help you to determine why your repository is not syncing with Dockstore.